CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v6.0.0 (2022-07-13)
```

Comment:
Note that for this project, we don't have a "bump" job, so this has to be added manually.  I haven't actually run the gem release jobs, as I wanted to do the PR first.  I'll do that after this is approved, with a corresponding PR (that will be small).